govcd/api_vcd.go

```diff
@@ -88,7 +88,7 @@ func (vcdCli *VCDClient) vcdauthorize(user, pass, org string) error {
 
 // NewVCDClient initializes VMware vCloud Director client with reasonable defaults.
 // It accepts functions of type VCDClientOption for adjusting defaults.
-func NewVCDClient(vcdEndpoint url.URL, insecure bool, options ...VCDClientOption) *VCDClient {
+func NewVCDClient(vcdEndpoint url.URL, insecure bool, timeout int, options ...VCDClientOption) *VCDClient {
```

Comment:
I would propose to set a reasonable default timeout (probably the 600 you're aiming for) and create a `WithTimeout(timeoutSeconds int) VCDClientOption`
similar to `func WithMaxRetryTimeout(timeoutSeconds int) VCDClientOption {`

That way you wouldn't break the `func NewVCDClient(` signature.

Reply:
done